client/app/styles/modules/_m-subscribers-confirm.scss

```diff
@@ -0,0 +1,30 @@
+// --------------------------------------------------
+// Module: Subscribers Confirmation Page
+// --------------------------------------------------
+
+.padding-top-4 {
+  padding-top: 4rem !important;
```

Comment:
I'll let it slide because I know this app's existing styles sometimes make it unavoidable but try to avoid `!important`.